PullRequestCreated/PullRequestCreated.ts

```diff
@@ -18,7 +18,7 @@ class PullRequestCreated extends OctokitAction {
       return;
     }
     let newTitle = pr.title.replace(/\s\s+/g, " ").trim();  // Mainly remove triple space between issue ID and title when copying from Jira
-    const linkedIssues = pr.title?.match(JIRA_ISSUE_PATTERN) || null;
+    const linkedIssues = pr.title.match(JIRA_ISSUE_PATTERN);
```

Comment:
Removed `?` as `pr.title` from the GH API should never be null. GH UI doesn't even allow to create PR with empty title.

`|| null` is no-op, as `.match` returns `null` by itself for zero matches.